apps/accounts/views.py

```diff
@@ -74,33 +73,37 @@ def verify_and_get_payload(token):
             # Some other token-related error
             raise TokenError("Token expired")
 
-   
-def OTP_DummyToken(user,purpose):
-    payload = {"email": user.email, "user_id": str(user.id), "user_type": user.user_type}
+
+def OTP_DummyToken(user, purpose):
+    payload = {
+        "email": user.email,
+        "user_id": str(user.id),
+        "user_type": user.user_type,
+    }
     token = GenerateToken.generate_dummy_jwt_token(payload)
-    
+
     # for old user
     if user.otp_secret:
         otp = OTP.generate_otp(user)
         user.save()
     # for new user
     else:
-        otp,secret = OTP.generate_secret_with_otp()
+        otp, secret = OTP.generate_secret_with_otp()
         user.otp_secret = secret
         user.save()
 
     # Send Email
-    if purpose=="verify":
-        subject="Verify your account"
+    if purpose == "verify":
+        subject = "Verify your account"
```

Comment:
can we create a constant.py, where we can store all the hardcoded or formatted strings we are using